rts/lra/lra-test/lra-test-basic/pom.xml

```diff
@@ -33,6 +33,12 @@
       <version>${version.thorntail}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.thorntail</groupId>
+      <artifactId>microprofile-config</artifactId>
+      <version>2.7.0.Final</version>
```

Comment:
the version would be good to be externalized to a property

Reply:
sure